### ISSUE TITLE
phovea:clone-repo fails silently on WSL 2

### ISSUE THREAD

**Issue by dvmichaelpuehringer** [phovea/generator-phovea#278]
* Release number or git hash: 2.0.1
* OS: Windows 10 (Build 19037) with Ubuntu 18.04 as WSL 2
* Environment (local or deployed): 

### Steps to reproduce

1. Install `generator-phovea` on a Windows 10 with WSL 10 (Ubuntu Distro)
2. Execute `yo phovea:setup-workspace ...` with a phovea product repo

### Observed behavior
```
mpuehringer@DESKTOP-F9HGI4F:~$ yo phovea:setup-workspace --skip=build <url>
? SSH clone Yes
create directory: <project_name>
running yo phovea:clone-repo

Just found a `.yo-rc.json` in a parent directory.
Setting the project root at: /home/mpuehringer
Error: Error: <project_name>/<project_name>_product/phovea_product.json: ENOENT: no such file or directory, open '<project_name>/<project_name>_product/phovea_product.json'
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: <project_name>/<project_name>_product/phovea_product.json: ENOENT: no such file or directory, open '<project_name>/<project_name>_product/phovea_product.json'
    at Object.openSync (fs.js:440:3)
    at Object.readFileSync (fs.js:342:35)
    at Object.readFileSync (/home/mpuehringer/.nvm/versions/node/v12.13.1/lib/node_modules/generator-phovea/node_modules/jsonfile/index.js:61:22)
    at /home/mpuehringer/.nvm/versions/node/v12.13.1/lib/node_modules/generator-phovea/generators/setup-workspace/index.js:193:27
Emitted 'error' event on Generator instance at:
    at Immediate.<anonymous> (/home/mpuehringer/.nvm/versions/node/v12.13.1/lib/node_modules/generator-phovea/node_modules/yeoman-generator/lib/index.js:415:18)
    at processImmediate (internal/timers.js:439:21) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '<project_name>/<project_name>_product/phovea_product.json'
}
```

### Expected behavior

* When commenting out https://github.com/phovea/generator-phovea/blob/master/generators/clone-repo/index.js#L66-L72, everything works as expected. Somehow, both of these calls lead to a silent (no proper error message) return. 
I suspect the https://github.com/phovea/generator-phovea/blob/master/utils/installedVersions.js#L18 (not investigated yet...) to be the culprit, as the script executes correctly up to this point: https://github.com/phovea/generator-phovea/blob/master/generators/check-node-version/index.js#L36.

**Comment by thinkh** [comment]
@dvmichaelpuehringer Where did you get generator v3.1.1? I suggest trying the develop branch, which contains this PR #266, which might fix this problem already.

**Comment by dvmichaelpuehringer** [comment]
@thinkh fixed version, original output was from `yo phovea --version` which prints the `yo` version apparently.

**Comment by keckelt** [comment]
I havn't had any problems in WSL2 so far with v1.1.1
2.0.1 fails here as well:
```
klaus@PC27651:/tmp/olol$ yo phovea:setup-workspace cohort_product -b develop
? SSH clone Yes
create directory: cohort
running yo phovea:clone-repo
Error: Error: cohort/cohort_product/phovea_product.json: ENOENT: no such file or directory, open 'cohort/cohort_product/phovea_product.json'
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: cohort/cohort_product/phovea_product.json: ENOENT: no such file or directory, open 'cohort/cohort_product/phovea_product.json'
    at Object.openSync (fs.js:451:3)
    at Object.readFileSync (fs.js:353:35)
    at Object.readFileSync (/home/klaus/.npm-global/lib/node_modules/generator-phovea/node_modules/jsonfile/index.js:61:22)
    at /home/klaus/.npm-global/lib/node_modules/generator-phovea/generators/setup-workspace/index.js:193:27
Emitted 'error' event on Generator instance at:
    at Immediate.<anonymous> (/home/klaus/.npm-global/lib/node_modules/generator-phovea/node_modules/yeoman-generator/lib/index.js:415:18)
    at processImmediate (internal/timers.js:439:21) {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'cohort/cohort_product/phovea_product.json'
}
```

**Comment by anita-steiner** [comment]
this is a bug on windows in 2.0.1 --> will be fixed in the next release

**Comment by thinkh** [comment]
I tried `yo phovea:setup-workspace ordino_product -b develop` in PowerShell and WSL 1 and it work with the generator develop branch. So it was fixed with PR #266 and will be available with the next release.

**Comment by thinkh** [comment]
@dvmichaelpuehringer and @keckelt We've just released [v3.0.0](https://github.com/phovea/generator-phovea/releases/tag/v3.0.0). Could you please update your generator with `npm install -g generator-phovea` and try it again?

**Comment by keckelt** [comment]
Works !
